card/mod/standard/set/all/rich_html/toolbar.rb

```diff
@@ -256,8 +256,7 @@ def autosaved_draft_link opts={}
   }.each do |viewname, viewtitle|
 
     view "#{viewname}_link" do
-      voo.title ||= viewtitle
-      link_to_view viewname, voo.title, class: "dropdown-item"
+      link_to_view viewname, viewtitle, class: "dropdown-item"
```

Comment:
this fixes the edit menu bug you saw.  I looked and couldn't find cases where it was useful to use voo.title.  In principle, it's an inappropriate use of that variable, anyway, imo.